superstyl/svm.py

```diff
@@ -246,3 +242,13 @@ def plot_coefficients(coefs, feature_names, current_class, top_features=10):
     plt.title("Coefficients for "+current_class)
     plt.savefig('coefs_' + current_class + '.png', bbox_inches='tight')
     # TODO: write them to disk as CSV files
+    # FOLLOW-UP: New code to write coefficients to disk as CSV 
+    # I also give back a message notifying of the file creation and showing the file name.
+    # First: pairing feature names with their coefficients
+    coefficients_df = pd.DataFrame({'Feature Name': feature_names, 'Coefficient': coefs})
```

Comment:
pandas is imported as `pandas` not as `pd` in this script.

Reply:
So `pandas.DataFrame`

Reply:
yup, done. Normalement tout bon, sauf codecov (dont je ne connais pas les manipulations). Dis moi si ça te va.